### PR TITLE
[SYCL] Removed handler::codeplay_host_task

### PR DESCRIPTION
**Pull request by maximdimakov**
handler::codeplay_host_task is deprecated and is removed in the patch https://github.com/intel/llvm/pull/4426
Signed-off-by: mdimakov <maxim.dimakov@intel.com>

### CODE REVIEW

File: SYCL/Basic/host-task-dependency.cpp [hunk 1]
```diff
@@ -98,11 +94,11 @@ template <bool UseSYCL2020HostTask> void Thread1Fn(Context *Ctx) {
         GeneratorAcc[Idx] = Idx;
     };
 
-    CGH.single_task<NameGen<class Gen, UseSYCL2020HostTask>>(GeneratorKernel);
+    CGH.single_task<NameGen<class Gen, true>>(GeneratorKernel);
```
Comment (dm-vodopyanov) [inline SYCL/Basic/host-task-dependency.cpp]: Does second template parameter still required in `NameGen`? Can it be removed?

**Comment by dm-vodopyanov** [comment]
> handler::codeplay_host_task is deprecated and will be removed

Does it mean that this patch removes handler::codeplay_host_task in this patch? If yes, it should be smth like: "and will be removed" -> "and is removed in this patch"

**Comment by maximdimakov** [comment]
@dm-vodopyanov I changed the comment

File: SYCL/Basic/host-task-dependency.cpp (hunk 1 above)
Comment (maximdimakov) [reply SYCL/Basic/host-task-dependency.cpp]: I removed it

**Review by dm-vodopyanov — APPROVED** [review]
LGTM

**Pull request merged**